go-controller/pkg/config/config.go

```diff
@@ -71,6 +71,9 @@ var (
 
 	// NbctlDaemon enables ovn-nbctl to run in daemon mode
 	NbctlDaemonMode bool
+
+	// PrivilegedMode needs ovnkube-node container to run with SYS_ADMIN capability by default.
+	PrivilegedMode = true
```

Comment:
If this was `UnprivilegedMode` instead then it would match the config flag, and wouldn't need to be initialized here, and could be set directly from the `cli.BoolFlag` declaration below rather than needing separate code in `ovnkube.go`.

As for the comment, assuming you go with `UnprivilegedMode`:

    // UnprivilegedMode allows ovnkube-node to run without SYS_ADMIN capability, by performing interface setup in the CNI plugin

(except don't you mean `NET_ADMIN` anyway?)

Reply:
Thanks for reviewing it.
I have updated the code.